src/main/java/pl/piekoszek/gorskimatches/equation/EquationRandomizer.java

```diff
@@ -17,4 +17,13 @@ public String randomEquation() {
         var quizzes = new ArrayList<>(equationGenerator.getAllSolutionsByQuiz().keySet());
         return quizzes.get(new Random().nextInt(quizzes.size()));
     }
+
+    public ArrayList<String> equationsForChallenge() {
+        var quizzes = new ArrayList<>(equationGenerator.getAllSolutionsByQuiz().keySet());
+        ArrayList<String> fiveQuizzesForChallenge = new ArrayList<>();
+        for (int i = 0; i < 5; i++) {
+            fiveQuizzesForChallenge.add(quizzes.get(new Random().nextInt(quizzes.size())));
```

Comment:
It's possible to select multiple times same quiz within same challenge, do we want that?